src/azure-account.api.d.ts

```diff
@@ -27,7 +27,7 @@ export interface AzureAccount {
 }
 
 export interface AzureSession {
-	readonly environment: AzureEnvironment;
+	readonly environment: Environment;
```

Comment:
I abbreviated this quite a bit, but you get the idea:

```
export declare class Environment {
    static readonly AzureCloud: {
    };

    /**
     * @deprecated use `AzureCloud` instead
     */
    static readonly Azure: {
    };


    static readonly ChinaCloud: {
    };

    /**
     * @deprecated use `ChinaCloud` instead
     */
    static readonly AzureChina: {
    };
}
```

You'll just have to copy the `AzureCloud` property over to an `Azure` property.